src/signals/incident-management/containers/IncidentDetail/components/StatusForm/components/ModalHeader/ModalHeader.tsx

```diff
@@ -1,15 +1,17 @@
 // SPDX-License-Identifier: MPL-2.0
 // Copyright (C) 2022 Gemeente Amsterdam
 import type { FC } from 'react'
-import styled from 'styled-components'
+import styled, { css } from 'styled-components'
 import { Button, Heading, themeColor, themeSpacing } from '@amsterdam/asc-ui'
 import { Close } from '@amsterdam/asc-assets'
 
-export const CloseButton = styled(Button)`
+export const CloseButton = styled(Button)<{ iconSize: number }>`
   border: none;
   padding: 0;
-  width: 20px;
-  height: 20px;
+  ${({ iconSize }) =>
+    css`
+      height: ${iconSize}px;
+    `}
```

Comment:
Can't `size` just be used as a prop on `CloseButton`? That should do the trick and shouldn't require having to set the height or the width explicitly with CSS.

Reply:
sure